src/data/indexData.ts

```diff
@@ -116,6 +121,34 @@ export const ORMGeneralLinkData = [
     url: '/orm/prisma-client/deployment',
     icon: 'fa-solid fa-book',
   },
+  {
+    title: `Accelerate`,
+    description:
+      'Set up connection pooling and global cache in your app with Accelerate.',
+    url: '/accelerate/getting-started',
+    icon: 'fa-solid fa-bolt',
+  },
+  {
+    title: `Pulse`,
+    description:
+      'React to changes in your database with Pulse.',
+    url: '/pulse/getting-started',
+    icon: 'fa-solid fa-signal-stream',
+  },
+  {
+    title: `Console`,
+    description:
+      'Explore how use our Console to efficiently collaborate with your team on the Prisma Data Platform.',
+    url: '/platform/concepts',
+    icon: 'fa-solid fa-chart-simple',
+  },
+  {
+    title: `Platform CLI commands`,
+    description:
+      'Learn how to access the Prisma Data Platform using the integrated commands in the Prisma CLI.',
```

Comment:
```suggestion
      'Learn how to access the Platform using the integrated commands in the Prisma CLI.',
```

We haven't shortened PDP to "Platform" a lot in the past, but I think here it'd fit well, also considering that a lot of the docs sections are called Platform.